{"level":"ERROR","location":"[On_error:261] order_ws","message":"Connection to remote host was lost.","timestamp":"2025-09-04 09:29:50,108+0530","service":"FyersDataSocket"}
{"level":"ERROR","location":"[<lambda>:1663] data_ws","message":"Connection to remote host was lost.","timestamp":"2025-09-04 09:55:48,820+0530","service":"FyersDataSocket"}
{"level":"ERROR","location":"[__parse_order_data:176] order_ws","message":"'org_ord_status'","timestamp":"2025-09-04 11:47:30,382+0530","service":"FyersDataSocket"}
{"level":"ERROR","location":"[On_error:261] order_ws","message":"Connection to remote host was lost.","timestamp":"2025-09-04 14:26:02,486+0530","service":"FyersDataSocket"}
{"level":"ERROR","location":"[<lambda>:1663] data_ws","message":"Connection to remote host was lost.","timestamp":"2025-09-04 15:27:33,625+0530","service":"FyersDataSocket"}
{"level":"ERROR","location":"[<lambda>:1663] data_ws","message":"Connection to remote host was lost.","timestamp":"2025-09-04 15:39:55,691+0530","service":"FyersDataSocket"}
{"level":"ERROR","location":"[<lambda>:1663] data_ws","message":"Connection to remote host was lost.","timestamp":"2025-09-04 15:41:04,319+0530","service":"FyersDataSocket"}
